packages/dev/core/src/Maths/math.vector.ts

```diff
@@ -1532,15 +1855,10 @@ export class Vector3 {
      * @returns the current updated Vector3
      */
     public minimizeInPlaceFromFloats(x: number, y: number, z: number): this {
-        if (x < this._x) {
-            this.x = x;
-        }
-        if (y < this._y) {
-            this.y = y;
-        }
-        if (z < this._z) {
-            this.z = z;
-        }
+        this._x = Math.min(x, this._x);
+        this._y = Math.min(y, this._y);
+        this._z = Math.min(z, this._z);
+        this._isDirty = true;

```

Comment:
I think you should revert to the old implementation, to avoid the 3 function calls. minimize/maximize are sometimes called quite a number of times (and both at the same time), so in the end it can make a difference to avoid the 6 function calls.

Reply:
Resolved in 4e38dc4